src/main/java/org/opentripplanner/routing/RoutingService.java

```diff
@@ -428,6 +429,8 @@ public void setDistanceBetweenElevationSamples(double distanceBetweenElevationSa
     /** {@link Graph#getTransitAlertService()} */
     public TransitAlertService getTransitAlertService() {return this.graph.getTransitAlertService();}
 
+    public RealtimeVehiclePositionService getVehiclePositionService() { return this.graph.getVehiclePositionService(); }
```

Comment:
(spacing inside braces is inconsistent with methods above)

Reply:
I've reformatted.